src/config.rs

```diff
@@ -1,29 +1,30 @@
 use std::fs;
 use std::io::ErrorKind::NotFound;
 
+use ansi_term::Color;
 use serde::{Deserialize, Serialize};
 
-use crate::memory::MemoryUnit;
+use crate::memory::Memory;
+use crate::shell::Shell;
 
 const BINARY_NAME: &str = env!("CARGO_PKG_NAME");
 
 #[derive(Serialize, Deserialize)]
 #[serde(default)]
 pub struct Config {
-    /// Unit used for memory usage. Possible values include KiB, MiB and GiB
-    pub mem_unit: MemoryUnit,
-    /// Whether to show memory usage as percentage
-    pub mem_percentage: bool,
-    /// Whether to show the full path of the shell
-    pub shell_path: bool,
+    /// Accent color in output
+    #[serde(skip)]
+    pub color: Color,
+    pub memory: Memory,
+    pub shell: Shell,
```

Comment:
lets have a space between these to make it more clear the skip is only for color